src/odb/src/codeGenerator/dbAccessPoint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "name": "dbAccessPoint",
+    "parent": "dbObject",
+    "fields": [
+      {
+        "name": "point_",
+        "type": "Point",
+        "flags":["cmpgt"]
+      },
+      {
+        "type": "dbId<_dbTechLayer>",
+        "name": "layer_",
+        "flags":["no-get","cmpgt"],
+        "parent": "dbTechLayer"
+      },
+      {
+        "type": "dbId<_dbLib>",
+        "name": "lib_",
+        "flags":["private","cmpgt"],
+        "parent": "dbLib"
+      },
+      {
+        "type": "dbId<_dbMaster>",
+        "name": "master_",
+        "flags":["private","cmpgt"],
+        "parent": "dbMaster"
+      },
+      {
+        "type": "dbId<_dbMPin>",
+        "name": "mpin_",
+        "flags":["private","cmpgt"],
+        "parent": "dbMPin"
+      },
+      {
+        "type": "dbId<_dbBPin>",
+        "name": "bpin_",
+        "flags":["private","cmpgt"],
+        "parent": "dbBPin"
+      },
+      {
+        "name": "accesses_",
+        "type": "dbVector<bool>",
```

Comment:
std::array<bool, 6> is smaller (vector<bool> is 40 bytes plus dynamic allocation in g++).